package.json

```diff
@@ -123,19 +124,19 @@
     "intersection-observer": "0.7.0",
     "is-ci": "^2.0.0",
     "is-docker": "1.1.0",
-    "jasmine-core": "^3.5.0",
-    "jasmine-spec-reporter": "^4.2.1",
-    "karma": "^4.4.1",
-    "karma-chrome-launcher": "^3.1.0",
-    "karma-coverage-istanbul-reporter": "^2.1.1",
-    "karma-jasmine": "^3.1.1",
-    "karma-jasmine-html-reporter": "^1.5.2",
+    "jasmine-core": "~3.6.0",
+    "jasmine-spec-reporter": "~5.0.0",
+    "karma": "~6.2.0",
+    "karma-chrome-launcher": "~3.1.0",
+    "karma-coverage-istanbul-reporter": "~3.0.2",
+    "karma-jasmine": "~4.0.0",
+    "karma-jasmine-html-reporter": "^1.5.0",
     "karma-junit-reporter": "^2.0.1",
     "karma-spec-reporter": "0.0.32",
     "lint-staged": "^9.4.2",
     "multimatch": "^4.0.0",
-    "ng-mocks": "^9.0.0",
-    "ng-packagr": "^9.0.3",
+    "ng-mocks": "^10.5.4",
+    "ng-packagr": "^11.2.4",
     "node-sass": "^4.13.1",
```

Comment:
I think `node-sass` can be bumped to 5.0 - which should allow us to bump our node version > 15

Reply:
It appears that `node-sass` is deprecated. This warning message is displayed when you run `npm start` after the Angular upgrade:

> 'node-sass' usage is deprecated and will be removed in a future major version. To opt-out of the deprecated behaviour and start using 'sass' uninstall 'node-sass'

So, perhaps I should just remove node-sass in this PR

Reply:
If possible, I think that would the best 👍  I found it to be very closely tied to specific node versions, which makes it a nice dependency to kill